openfe/protocols/openmm_rfe/equil_rfe_methods.py

```diff
@@ -256,6 +256,15 @@ def __init__(self, **data):
         if any(len(pur_list) > 2 for pur_list in self.data.values()):
             raise NotImplementedError("Can't stitch together results yet")
 
+    @staticmethod
+    def compute_mean_estimate(dGs:list[dict]):
+        u = dGs[0].u
+        # convert all values to units of the first value, then take average of magnitude
+        # this would avoid a screwy case where each value was in different units
```

Comment:
Might just be my sensibilities, and I know it's a historical comment, but I'm not super fond of this comment - would be happy to see it reworded.